setup.sh

```diff
@@ -19,25 +19,28 @@ function getLatestK8s {
    echo $latestVersion
 }
 
-# Environment variables / positional parameters and defaults. 
+# Environment variables / positional parameters and defaults.
+
+# Add ~/bin to PATH for the script only 
+export PATH=$PATH:~/bin
 
 # TODO make $1 and $2 required?
-keyName=$1
+read -p "Enter the key name used to sign and verify: "  keyName
 
-keySubjectName=$2
+read -p "Enter the key subject name: "  keySubjectName
 
-rgName=$3
+read -p "Enter the resource group to deploy into: "  rgName
```

Comment:
I like the simplicity a read prompt offers, but the user may thing they _have_ to enter one after having to enter the previous two fields. I propose leaving positional parameters in place where a user could over ride if they wish, but it's not necessary to run the script.